devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -142,7 +143,11 @@ private createMeterEventMap(cmd) {
 
 def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelEndPointReport cmd, ep = null) {
 	if(!childDevices) {
-		addChildSwitches(cmd.endPoints)
+		if (zwaveInfo.mfr.equals("027A") && zwaveInfo.model.equals("A004")) {
```

Comment:
Please use helper function to identify the devices. It helps in readability. 

Also, This change should have a comment that documents the reason behind this change.

Reply:
I think @kianooshST was asking for a helper function like `isZoozZenStripV2()` which would check the `mfr` and `model`. For the comment something like `cmd.endPoints includes the USB ports but we don't want to expose them as child devices since they cannot be controlled so hardcode to just include the outlets` would be more descriptive (please verify that's accurate too).